package.json

```
@@ -257,16 +254,21 @@
     "source-map-support": "^0.5.6",
     "ts-node": "^7.0.0",
     "tslint": "^5.10.0",
-    "typescript": "^2.9.1",
+    "typescript": "^3.0.0",
+    "vscode": "^1.1.32",
     "vscode-test": "^1.0.2"
   },
   "dependencies": {
+    "@octokit/rest": "^16.28.4",
     "applicationinsights": "^1.0.5",
     "graphlib-dot": "^0.6.2",
     "lodash": "^4.17.11",
     "minimatch": "^3.0.4",
+    "tar": "^4.4.10",
+    "typed-rest-client": "^1.5.0",
     "uuid": "^3.3.2",
     "viz.js": "^1.8.2",
+    "vscode-languageclient": "^5.2.1",
     "vscode-uri": "^1.0.5"
```

For now we may want to have this capped at `1.0.6` - anything beyond that, including the `2.x` releases, causes tests to break. Or maybe there is a simple test tweak we can make...

```suggestion
    "vscode-uri": "^1.0.5 <=1.0.6"
```

---

master now uses `"vscode-uri": "^1.0.8"` and the tests pass but I haven't tested the plugin more than just running units and integs, I had to fix some tests to make it pass though

---

Cool, I've merged master into this work so hopefully all set now.